custom_components/plugwise_usb/select.py

```
@@ -0,0 +1,151 @@
+    @property
+    def current_option(self) -> str:
+        """Return the currently selected option."""
+        return self._current_option
+
+    async def async_select_option(self, option: str) -> None:
+        """Change to the selected entity option."""
+        value = self.entity_description.options[option]
+        await self.async_select_fn(value)
+        self._current_option = option
+        self._attr_current_option = option
+        self.async_write_ha_state() 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)